src/utils/createStylingFromTheme.js

```diff
@@ -250,10 +257,11 @@ const getSheetFromColorMap = map => ({
 
   previewHeader: {
     padding: '5px 10px',
+    'flex-basis': '0px',
     'align-items': 'center',
     'border-bottom-width': '1px',
     'border-bottom-style': 'solid',
-    'min-height': '30px',
+    'min-height': '40px',
```

Comment:
@zalmoxisus because I using `min-height` and `flex-basis` before, 30px will get unexpected result, but we should keep 30px for change to `flex: '0 0 px'`, it's my fault, I'll update it.